### PR TITLE
View/play selected file

### PR DESCRIPTION
**Pull request by MelissaRoman06**

### CODE REVIEW

File: src/main/java/com/fundation/logic/common/FileInfo.java [hunk 1]
```diff
@@ -124,4 +124,20 @@ public static String getMimeType(File file) {
         }
         return null;
     }
+
```
Comment (TheYisusGithub) [inline src/main/java/com/fundation/logic/common/FileInfo.java:131]: Insert comment in this line.

File: src/main/java/com/fundation/logic/common/FileInfo.java [hunk 2]
```diff
@@ -124,4 +124,20 @@ public static String getMimeType(File file) {
...
+    public static boolean isVideo(String path){
+        File file = new File(path);
+        if(getMimeType(file).contains("video")){
+            return true;
+        }
+        return false;
+    }
+
```
Comment (TheYisusGithub) [inline src/main/java/com/fundation/logic/common/FileInfo.java:144]: Please comment this method.

File: src/main/java/com/fundation/logic/view/PlayerFrame.java [hunk 3]
```diff
@@ -72,28 +75,23 @@ private void initComponent(String path) {
         volumeSlider = new JSlider(JSlider.VERTICAL, 0, 100, volumeLevel);
```
Comment (TheYisusGithub) [inline src/main/java/com/fundation/logic/view/PlayerFrame.java]: Magical number in this line.

File: src/main/java/com/fundation/logic/view/PlayerFrame.java [hunk 4]
```diff
@@ -72,28 +75,23 @@ private void initComponent(String path) {
...
-        buttonListener = new ButtonListener(volumeLevel);
+        buttonListener = new PlayerButtonListener(volumeLevel);
     }
 
     /**
-     * Set all PlayerJFrame components
+     * Set all PlayerFrame components
      */
     private void initSetting() {
         setBounds(new Rectangle(80, 100, 800, 540));
```
Comment (TheYisusGithub) [inline src/main/java/com/fundation/logic/view/PlayerFrame.java]: In this line you have use magical number.

File: src/main/java/com/fundation/logic/view/PlayerFrame.java [hunk 5]
```diff
@@ -72,28 +75,23 @@ private void initComponent(String path) {
...
-            System.out.println("Player icons cannot be found..");
-        }
-
+        setTitle(fileToBePlayed.getName());
         add(bottomPanel, BorderLayout.SOUTH);
+        playButton.setIcon(new ImageIcon("resources/Images/Play.png"));
         playButton.setPreferredSize(new Dimension(35, 35));
         bottomPanel.add(playButton);
+        pauseButton.setIcon(new ImageIcon("resources/Images/Pause.png"));
         pauseButton.setPreferredSize(new Dimension(35, 35));
```
Comment (TheYisusGithub) [inline src/main/java/com/fundation/logic/view/PlayerFrame.java]: Please show this line magical number.

File: src/main/java/com/fundation/logic/view/PlayerFrame.java [hunk 6]
```diff
@@ -72,28 +75,23 @@ private void initComponent(String path) {
...
         bottomPanel.add(pauseButton);
+        stopButton.setIcon(new ImageIcon("resources/Images/Stop.png"));
         stopButton.setPreferredSize(new Dimension(35, 35));
```
Comment (TheYisusGithub) [inline src/main/java/com/fundation/logic/view/PlayerFrame.java]: Magical number in this line.

File: src/main/java/com/fundation/logic/view/PlayerFrame.java [hunk 7]
```diff
@@ -72,28 +75,23 @@ private void initComponent(String path) {
...
         bottomPanel.add(stopButton);
         progressBar.setPreferredSize(new Dimension(610, 20));
```
Comment (TheYisusGithub) [inline src/main/java/com/fundation/logic/view/PlayerFrame.java]: In the progress bar change the magical numbers.

File: src/main/java/com/fundation/logic/view/TableResult.java [hunk 8]
```diff
@@ -44,8 +49,26 @@ public void addResult(String path, String extension, Float size, Date date, Stri
...
+
+    /**
+     * Listen to right click to play the selected row's file.
+     */
+    public void initListen() {
+        this.addMouseListener(new MouseAdapter() {
+            public void mouseClicked(MouseEvent me) {
+                if (me.getButton() == MouseEvent.BUTTON3) {
+                    int row = getSelectedRow();
+                    String filePath = (String) model.getValueAt(row, 0);
```
Comment (TheYisusGithub) [inline src/main/java/com/fundation/logic/view/TableResult.java]: Don't use magical numbers.

File: src/main/java/com/fundation/logic/common/FileInfo.java (hunk 1 above)
Comment (MelissaRoman06) [reply src/main/java/com/fundation/logic/common/FileInfo.java:131]: done

File: src/main/java/com/fundation/logic/common/FileInfo.java (hunk 2 above)
Comment (MelissaRoman06) [reply src/main/java/com/fundation/logic/common/FileInfo.java:144]: done

File: src/main/java/com/fundation/logic/view/PlayerFrame.java (hunk 3 above)
Comment (MelissaRoman06) [reply src/main/java/com/fundation/logic/view/PlayerFrame.java]: done

File: src/main/java/com/fundation/logic/view/PlayerFrame.java (hunk 4 above)
Comment (MelissaRoman06) [reply src/main/java/com/fundation/logic/view/PlayerFrame.java]: Magical numbers changed to constants

File: src/main/java/com/fundation/logic/view/PlayerFrame.java (hunk 6 above)
Comment (MelissaRoman06) [reply src/main/java/com/fundation/logic/view/PlayerFrame.java]: magical numbers changed to constants

File: src/main/java/com/fundation/logic/view/PlayerFrame.java (hunk 7 above)
Comment (MelissaRoman06) [reply src/main/java/com/fundation/logic/view/PlayerFrame.java]: Magical numbers changed to constants

File: src/main/java/com/fundation/logic/view/TableResult.java (hunk 8 above)
Comment (MelissaRoman06) [reply src/main/java/com/fundation/logic/view/TableResult.java]: ok

File: src/main/java/com/fundation/logic/view/PlayerFrame.java (hunk 5 above)
Comment (MelissaRoman06) [reply src/main/java/com/fundation/logic/view/PlayerFrame.java]: changed to constants

File: src/main/java/com/fundation/logic/common/FileInfo.java (hunk 2 above)
Comment (MelissaRoman06) [reply src/main/java/com/fundation/logic/common/FileInfo.java:144]: Comments added

**Pull request merged**